<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioEditor</class>
 <widget class="QDialog" name="ScenarioEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Scenario</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,1,0,0,0,0,0,0">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QComboBox" name="rewardFunc">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="rewardFuncUse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Use script</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="playerId">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QComboBox" name="variableName"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLineEdit" name="doneNodeName">
     <property name="text">
      <string>subnodeName</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="addDone">
     <property name="text">
      <string>Add End Condition</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QPushButton" name="reloadScripts">
     <property name="text">
      <string>Reload Scripts</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="addReward">
     <property name="text">
      <string>Add Reward</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="addDoneNode">
     <property name="text">
      <string>Add Sub-Condition</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="guess">
     <property name="text">
      <string>Guess</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="loadScript">
     <property name="text">
      <string>Load Script</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4" rowspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QComboBox" name="doneFunc">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="doneFuncUse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Use script</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="reward">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="rewardList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="doneList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>961</width>
        <height>24</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScenarioEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScenarioEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doneFuncUse</sender>
   <signal>toggled(bool)</signal>
   <receiver>doneFunc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rewardFuncUse</sender>
   <signal>clicked(bool)</signal>
   <receiver>rewardFunc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
